cwms-data-api/src/test/resources/cwms/cda/data/dto/stream_reach.json

```diff
@@ -0,0 +1,22 @@
+{
+  "reach-id": {
+    "office-id": "Office123",
+    "location-id": "Reach123"
+  },
+  "stream-id": {
+    "office-id": "Office123",
+    "location-id": "Stream123"
+  },
+  "upstream-station": 10.0,
+  "downstream-station": 20.0,
+  "comment": "This is a comment for the stream reach.",
+  "downstream-location-id": {
+    "office-id": "Office123",
+    "location-id": "Downstream123"
+  },
+  "upstream-location-id": {
+    "office-id": "Office123",
+    "location-id": "Upstream123"
+  },
+  "configuration-id": "Config123"
```

Comment:
configuration has an office

Reply:
updated configuration ID to LocationIdentifier (will rename locationIdentifier to CwmsIdentifier in a new class)